static/styles/zulip.scss

```diff
@@ -612,16 +607,13 @@ td.pointer {
     top: -3px;
 }
 
-.status-time {
-    top: 8px !important;
+.include-sender .message_controls {
+    line-height: 1;
```

Comment:
Why do we need all these `line-height: 1` values?

Reply:
Yeah, in the places there is, since somewhere for the parent it was changed.